packages/firestore/src/local/simple_db.ts

```diff
@@ -66,7 +66,14 @@ export class SimpleDb {
       // suggests IE9 and older WebKit browsers handle upgrade
       // differently. They expect setVersion, as described here:
       // https://developer.mozilla.org/en-US/docs/Web/API/IDBVersionChangeRequest/setVersion
-      const request = window.indexedDB.open(name, version);
+
+      let request: IDBOpenDBRequest;
+      // Support Web Workers, which don't have `window` global variable.
+      if (indexedDB !== undefined) {
```

Comment:
AFAIK the only way to safely check for the existence of a global is `typeof indexedDB !== 'undefined'`

https://imgur.com/a/28vgzqX

So your code here only works because we've already (safely) checked for the existence of the global in isAvailable().